pkg/kv/kvserver/raft_transport.go

```diff
@@ -301,6 +301,11 @@ func NewRaftTransport(
 	return t
 }
 
+// Metrics returns metrics tracking this transport.
+func (t *RaftTransport) Metrics() *RaftTransportMetrics {
```

Comment:
Alternatively, can do a `Register(r *metric.Register)` method, to avoid exposing the metrics.

Reply:
Leaving as is.